### PR TITLE
Update post-aggregations.md

### PR DESCRIPTION
**Pull request by SpyderRivera**
I think this is more clear. I am not sure how multiplying by 100 is involved in averaging...I assume it was a copy paste typo

### CODE REVIEW

File: docs/content/querying/post-aggregations.md [hunk 1]
```diff
@@ -162,14 +162,35 @@ The format of the query JSON is as follows:
...
     "fn"     : "*",
     "fields" : [
        { "type"   : "arithmetic",
-         "name"   : "div",
+         "name"   : "ratio",
          "fn"     : "/",
          "fields" : [
-           { "type" : "fieldAccess", "name" : "tot", "fieldName" : "tot" },
-           { "type" : "fieldAccess", "name" : "rows", "fieldName" : "rows" }
+           { "type" : "fieldAccess", "name" : "part", "fieldName" : "part" },
```
Comment (jihoonson) [inline docs/content/querying/post-aggregations.md]: ```fieldName``` should be the name of an aggregator. Maybe you should add one more aggregator like ```{ "type" : "doubleSum", "name" : "part", "fieldName" : "part" }```.

**Comment by leventov** [comment]
@SpyderRivera thanks for contribution, please sign CLA: http://druid.io/community/cla.html

**Comment by b-slim** [comment]
@SpyderRivera any chance you can finish this up?

File: docs/content/querying/post-aggregations.md (hunk 1 above)
Comment (SpyderRivera) [reply docs/content/querying/post-aggregations.md]: I see

**Pull request merged**